pkg/test/secrethelper.go

```diff
@@ -41,14 +43,14 @@ func HelperGetRegisterDocument() (*register.RegisterDocument, *register.Issuer,
 	return registerDocument, issuer, ValidKeyPair
 }
 
-func SetupIdentitiesForAuth(resolver register.ResolverClient, control bool, auth bool) (register.RegisteredIdentity, register.RegisteredIdentity) {
+func SetupIdentitiesForAuth(ctx context.Context, resolver register.ResolverClient, control bool, auth bool) (register.RegisteredIdentity, register.RegisteredIdentity) {
 	userSecret, _ := crypto.NewDefaultKeyPairSecrets(ValidBip39Seed32B, "iotics/0/user/00")
 	userKeypair, _ := crypto.GetKeyPair(userSecret)
-	userIdentity, _, _ := advancedapi.CreateNewIdentityAndRegister(resolver, identity.User, userKeypair, "#user", true)
+	userIdentity, _, _ := advancedapi.CreateNewIdentityAndRegister(ctx, resolver, identity.User, userKeypair, "#user", true)
```

Comment:
Given we ignore all errors - Is there any point in passing in the context into this test func? Might as well define a `context.TODO()` at the start instead maybe?

Reply:
true

Reply:
https://github.com/Iotic-Labs/iotics-identity-go/pull/46/commits/6a622a694ffdd3b3a460e26a47373170fcccbcf2